packages/create-cloudflare/src/frameworks/solid/templates.ts

```diff
@@ -1,9 +1,12 @@
 export const viteConfig = `
-import solid from "solid-start/vite";
-import { defineConfig } from "vite";
-import cloudflare from "solid-start-cloudflare-pages";
+import { defineConfig } from "@solidjs/start/config";
 
 export default defineConfig({
-  plugins: [solid({ adapter: cloudflare({}) })],
+    // @ts-ignore
+    start: {
+        server: {
+            preset: "cloudflare-pages",
```

Comment:
in `server` maybe we should add this:
```ts
rollupConfig: {
  external: ["node:async_hooks"]
},
```

as it might be necessary 🤔 

@ryansolid what would you advise? 🙏

Reply:
Sounds like we should include this.

Reply:
Yes include it. Unfortunately because node compat isn't default unjs will try to polyfill and it screws things up if node compat is on. So here we are.

Reply:
Added, thanks a lot for the insight @ryansolid 🙂👍 

(sorry for the delay in updating this, I've got sidetracked in other things 😓)